app/views/accounts/show.txt.erb

```diff
@@ -0,0 +1,3 @@
+<% @account.statuses.each do |status| %>
+<%= status.created_at %>	<%= extract_status_plain_text status %>
```

Comment:
The time string is RFC 3339, but I believe `status.created_at` will output ISO 8601. [They're often the same](https://ijmacd.github.io/rfc3339-iso8601/), but it might be nice to be clear:

```suggestion
<%= status.created_at.rfc3339 %>	<%= extract_status_plain_text status %>
```

Reply:
Definitely needs to be RFC3339 or clients/parsers w=won't parse it 👌